pkgs/applications/version-management/gogs/default.nix.patch

```diff
@@ -0,0 +1,22 @@
+2,3c2
```

Comment:
`go2nix` ought to create a trivially overridable expression. That way, the current `default.nix` becomes `generated.nix`, and then a new `default.nix` do something like `callPackage ./generated.nix { postInstall = "..."; }`.

You can see here for an example of how the haskell package builder supports overriding: 

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/haskell-modules/generic-builder.nix

It's not ideal to have to apply some patch left in the source tree each time the package is updated.

Sorry for being terse - gotta get back to work. If you have an questions, feel free to ask here or ping me on IRC.